src/model_cooling_heating.c

```diff
@@ -63,7 +62,8 @@ double cooling_recipe(const int gal, const double dt, struct GALAXY *galaxies, c
 		coolingGas = 0.0;
     }
 
-	assert(coolingGas >= 0.0);
+	XASSERT(coolingGas >= 0.0, -1,
```

Comment:
I unfortunately think that all our handling of errors "properly" is only included in the non-Physics modules.  So I think converting these to use the new handling of `return EXIT_FAILURE` is beyond the scope...

Reply:
Yup!  Fixing the `ABORT` or `ASSERT` within the `model_` source files is out of scope.